code/game/objects/effects/decals/contraband.dm

```diff
@@ -79,20 +79,12 @@
 //############################## THE ACTUAL DECALS ###########################
 
 /obj/structure/sign/poster
-	name = "poster"
```

Comment:
I'm not quite sure why this was unset.

Reply:
Because that's never used. It used to slap the initial name into the generated name. It doesn't anymore.